packages/bridge-ui-v2/src/components/Bridge/Recipient.svelte

```diff
@@ -0,0 +1,123 @@
+<script lang="ts">
```

Comment:
I saw you moved a couple of components out of their folders, but left others. Is there a reason? Now it seems a bit inconsistent, some have their export through an `index.ts`, some don't. Some are in a folder, some are not. I think we should be consistent.

Reply:
Which components do you refer to?

Reply:
Let's sync offline